src/chttpd/src/chttpd_misc.erl

```diff
@@ -146,14 +150,31 @@ all_dbs_callback({row, Row}, #vacc{resp = Resp0} = Acc) ->
             {ok, Resp1} = chttpd:send_delayed_chunk(Resp0, [Prepend, ?JSON_ENCODE(DbName)]),
             {ok, Acc#vacc{prepend = ",", resp = Resp1}}
     end;
-all_dbs_callback(complete, #vacc{resp = Resp0} = Acc) ->
+all_dbs_info_callback({row, Row}, #vacc{resp = Resp0} = Acc)
+  when Acc#vacc.req#httpd.path_parts =:= [<<"_dbs_info">>] ->
+    Prepend = couch_mrview_http:prepend_val(Acc),
+    DbName = couch_util:get_value(id, Row),
+    case chttpd_util:get_db_info_helper(DbName) of
+        {ok, DbInfo} ->
+            Chunk = [Prepend, ?JSON_ENCODE({[{key,DbName}, {info, {DbInfo}}]})],
+            {ok, Resp1} = chttpd:send_delayed_chunk(Resp0, Chunk),
+            {ok, Acc#vacc{prepend = ",", resp = Resp1}};
+        {error, database_does_not_exist} ->
+            {ok, Acc#vacc{prepend = ",", resp = Resp0}};
```

Comment:
We should verify that a "," here would work. I suspect it might not. If we get a deleted db first, then on next row emit wouldn't we end up with `[, {...}]`?

Reply:
Good eye! Removed prepend.
Thanks for the review.